sql/dd/upgrade/schema.cc

```diff
@@ -155,7 +155,7 @@ bool migrate_schema_to_dd(THD *thd, const char *dbname) {
   // Disable autocommit option
   Disable_autocommit_guard autocommit_guard(thd);
 
-  if (dd::create_schema(thd, schema_name, schema_charset)) {
+  if (dd::create_schema(thd, schema_name, schema_charset, nullptr)) {
```

Comment:
In 5.6, we store the db_metadata information in the db_opt file using the keyword `db-metadata`. Since there are in-place upgrade paths planned to 8.0, it should be fine to have nullptr here.